app/controllers/placements/providers/placements_controller.rb

```diff
@@ -19,7 +19,10 @@ def index
   end
 
   def show
-    @placement = Placement.find(params[:id]).decorate
+    @placement = policy_scope(
+      Placement.all,
+      policy_scope_class: Placements::Provider::PlacementPolicy::Scope,
+    ).find(params.require(:id)).decorate
```

Comment:
This one might be trickier because `#index` uses a query object – but I wonder if [Active Record's merge method](https://api.rubyonrails.org/classes/ActiveRecord/SpawnMethods.html#method-i-merge) would work? 🤔 

```ruby
class PlacementsController
  def index
    query = Placements::PlacementsQuery.call(params: query_params)
    @pagy, @placements = pagy(placements.merge(query))
  end

  def show
    @placement = placements.find(params.require(:id))
    @school = @placement.school
  end

  private

  def placements
    policy_scope(
      Placement.all,
      policy_scope_class: Placements::Provider::PlacementPolicy::Scope,
    )
  end
end
```